modules/tracking/tutorials/tutorial_introduction_to_tracker.markdown

```diff
@@ -0,0 +1,75 @@
+Introduction to OpenCV Tracker {#tutorial_introduction_to_tracker}
+==============================
+
+Goal
+----
+
+In this tutorial you will learn how to
+
+-   Create a tracker object.
+-   Using the roiSelector function to select ROI from a given image.
+-   Track a specific region from a given image.
+
+Source Code
+-----------
+
+@include tracking/samples/tutorial_introduction_to_tracker.cpp
+
+Explanation
+-----------
+
+-#  **Set up the input video**
+
+    @snippet tracking/samples/tutorial_introduction_to_tracker.cpp help
+
+    In this tutorial, you can choose between video or list of images for the program input.
+    As written in the help, you should specify the input video as parameter of the program.
+    If you want to use image list as input, the image list should have formatted numbering
+    as shown in help. In the help, it means that the image fils are numbered with 4 digits
+    (e.g. the file naming will be 0001.jpg, 0002.jpg, and so on).
+
+    You can find samples video in Itseez/opencv_extra/testdata/cv/tracking
```

Comment:
yes, in that folder there are 3 videos, maybe was added for previous development of tracker algorithm